tests/reducer-test.js

```diff
@@ -0,0 +1,85 @@
+const tape = require('tape');
+
+const reducer = require('./../app/reducers/index.js').default;
+const types = require('./../app/action_types.js');
+const steps = require('./../app/steps.js');
+
+const defaultState = {
+  goals: [],
+  step: steps.GOALS_LIST,
+  previousStep: null,
+  newGoal: {},
+  currentGoal: null,
+};
+
+tape('test reducer step_add_goal: step and previousStep changed', (t) => {
+
+  const initialState = defaultState;
+  const actionStepAddGoal = {
+    type: types.STEP_ADD_GOAL,
+  };
+
+  t.equal(reducer(initialState, actionStepAddGoal).step, steps.ADD_GOAL, "step add goal sets correct step");
```

Comment:
Can you break these onto multiple lines?
```js
t.equal(
  reducer(initialState, actionStepAddGoal).step,
  steps.ADD_GOAL,
  "step add goal sets correct step"
);
```
Other wise the diffs get really ugly